libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -4763,6 +4767,35 @@ void GCS_MAVLINK::send_set_position_target_global_int(uint8_t target_system, uin
             0,0);   // yaw, yaw_rate
 }
 
+// returns true if all battery instances were reported
+bool GCS_MAVLINK::send_smart_battery_info()
+{
+#if HAL_SMART_BATT_LOG_ENABLED
+    const AP_BattMonitor &battery = AP::battery();
+
+    for(uint8_t i = 0; i < battery.num_instances(); i++) {
```

Comment:
nitpick: we normally have a space between "for" and the following bracket.

Reply:
fixed